src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        ColorSupplier colloSuplier = new ColorSupplier();
+        int namber = new Random().nextInt(100);
```

Comment:
1. random should be `private final` class-level field
2. typo in 'namber'
3. 100 is a magic number, should be constant
```suggestion
        int number = random.nextInt(MAX_NUMBER);
```